app/mender.go

```diff
@@ -275,6 +276,35 @@ func verifyArtifactDependencies(
 	return nil
 }
 
+type UpdateResponse struct {
```

Comment:
Update response parsing has now been moved to `mender.go`. Which is really our only option AFAICT.